project-base/storefront/components/Basic/Icon/UserIcon.tsx

```diff
@@ -1,9 +1,10 @@
-export const UserIcon: SvgFC = (props) => (
-    <svg {...props} viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg">
+export const UserIcon: SvgFC<{ isFull: boolean }> = ({ isFull, ...props }) => (
+    <svg {...props} fill={isFull ? 'currentColor' : 'none'} viewBox="0 0 25 25" xmlns="http://www.w3.org/2000/svg">
         <path
-            d="M255.992 315.584c-59.93 0-111.747 14.714-152.185 45.576C63.368 392.022 35.587 438.08 20 495.766l50.822 13.732c13.368-49.474 35.166-83.777 64.924-106.488 29.758-22.711 68.752-34.781 120.246-34.781 51.492 0 90.487 12.074 120.246 34.789 29.76 22.714 51.558 57.021 64.926 106.482l50.822-13.736C476.4 438.092 448.62 392.035 408.182 361.17c-40.437-30.865-92.257-45.586-152.19-45.586zm0-313.084c-72.376 0-131.613 59.237-131.613 131.613 0 72.377 59.237 131.616 131.613 131.616 72.377 0 131.615-59.24 131.615-131.616S328.37 2.5 255.992 2.5zm0 52.646c43.925 0 78.969 35.043 78.969 78.967 0 43.925-35.044 78.969-78.969 78.969s-78.967-35.044-78.967-78.969c0-43.924 35.042-78.967 78.967-78.967z"
-            fill="currentColor"
-            overflow="visible"
+            d="M12.5 14.788a5 5 0 1 0 0-10 5 5 0 0 0 0 10Zm0 0c-4.418 0-8 2.686-8 6m8-6c4.418 0 8 2.686 8 6"
+            stroke="currentColor"
+            strokeLinecap="round"
+            strokeWidth="2"
         />
     </svg>
```

Comment:
I noticed that the user icon appears similar to the others. Could you please double-check this?
![image](https://github.com/user-attachments/assets/c71b0e93-71bb-4bdb-8abb-4456ba05319e)

Reply:
True fixed.